container-conf/build.sh

```diff
@@ -10,23 +10,21 @@ build_is_public=1
 build_dockerfile_aux="USER $build_run_user"'
 ENV NVIDIA_VISIBLE_DEVICES all
 ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
-ENV NVIDIA_REQUIRE_CUDA "cuda>=10.0 brand=tesla,driver>=410"'
+ENV NVIDIA_REQUIRE_CUDA "cuda>=11.1 brand=tesla,driver>=410"'
 # use previous command
 build_docker_cmd=
 
 build_as_root() {
     umask 022
     cd "$build_guest_conf"
-    dnf config-manager --add-repo http://developer.download.nvidia.com/compute/cuda/repos/fedora29/x86_64/cuda-fedora29.repo
-    build_yum install cuda-toolkit-10-1
+    dnf config-manager --add-repo http://developer.download.nvidia.com/compute/cuda/repos/fedora32/x86_64/cuda-fedora32.repo
+    build_yum install cuda-toolkit-11-1
     cat > /etc/ld.so.conf.d/rs-cuda.conf <<'EOF'
-/usr/local/cuda-10.2/targets/x86_64-linux/lib
-/usr/local/cuda/extras/CUPTI/lib64
```

Comment:
Are you sure CUPTI is not required/installed? I see that it is still being developed.

Reply:
The CUPTI libs now seem to be in /usr/local/cuda-11.1/targets/x86_64-linux/lib so just that line covers everything.